modules/telco-ran-414-whats-new-ref-design.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-design-spec.adoc
+
+:_content-type: CONCEPT
+[id="telco-ran-whats-new-ref-design_{context}""]
+= What's new in {product-title} {product-version} for telco RAN
+
+The following new features included in {product-title} {product-version} are leveraged by the telco RAN DU reference design.
+
+.New features for telco RAN in {product-title} {product-version}
+[cols="1,3", options="header"]
+|====
+|Feature
+|Description
+
+//CNF-7078
+|Intel e810 NIC as PTP Grandmaster clock
+a|You can use the Intel E810-XXV-4T NIC as a GNSS-sourced grandmaster clock.
+The NIC is automatically configured by the PTP Operator with the E810 hardware plugin.
+This feature is scheduled for an upcoming 4.14 z-stream release.
+
+* link:https://docs.openshift.com/container-platform/4.14/networking/ptp/configuring-ptp#configuring-linuxptp-services-as-grandmaster-clock_configuring-ptp[Configuring linuxptp services as a grandmaster clock]
+
+//CNF-6527
+|PTP Operator hardware specific functionality plugin
+a|A new E810 NIC hardware plugin is now available in the PTP Operator.
+You can use the E810 plugin to configure the NIC directly.
+This feature is scheduled for an upcoming 4.14 z-stream release.
+
+* link:https://docs.openshift.com/container-platform/4.14/networking/ptp/configuring-ptp#nw-ptp-wpc-hardware-pins-reference_configuring-ptp[Intel Westport Channel E810 hardware configuration reference]
```

Comment:
The first column speaks about generic E810, while the link looks very specific to WPC